docs/catalogue/cat_cohort-data-manager.md

```diff
@@ -147,6 +162,8 @@ that has to be repeated when filling out repeated variables. This sheet is optio
 
 | *Column name* | *Description* | *Remarks* |
 | --- | --- | --- |
+| dataDictionary.resource \* | Data dictionary that this variable belongs to. | Fill out your cohort pid. |
+| dataDictionary.version \* |  Version of the data dictionary that this variable belongs to. | |
 | table \* | Table name. | e.g. core |
 | name \* | Variable name. | e.g. height\_1 |
 | isRepeatOf.table \* | Table that contains the variable that is repeated. | Tables must be predefined in the _
```

Comment:
markup table breaks after 'in the _'
also for isRepeatOf.name *, collectionEvent